frontend/public/locales/en/bma.json

```diff
@@ -28,7 +28,8 @@
   "editBareMetalAsset.title": "Edit bare metal asset",
   "editBareMetalAsset.button.submit":"Apply",
 
-  "bareMetalAsset.emptyState.title":"No bare metal assets found",
+  "bareMetalAsset.emptyState.title":"You don't have any bare metal assets.",
+  "bareMetalAsset.emptyState.subtitle": "Click the <bold>Create bare metal asset</bold> or <bold>Import bare metal assets</bold> button to create your resource",
```

Comment:
@showeimer 
I found that I was misinterpreting Joy's design. The newline she had was the space between the title and the message of the emptyState, I thought it was all part of the message. No need for a <br>.